sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -367,6 +367,9 @@ object functions {
    */
   def collect_set(columnName: String): Column = collect_set(Column(columnName))
 
+  private[spark] def collect_top_k(e: Column, num: Int, reverse: Boolean): Column =
```

Comment:
nit: shall we make it public ? It might be a useful function.

We don't need to  do it in this PR.

Reply:
I don't know, I also think it's useful and may further use it in Pandas-API-on-Spark.
But I don't know whether it is suitable to be public @cloud-fan @HyukjinKwon

Reply:
Let's keep it private for now.